lib/utils/ember.js

```diff
@@ -89,11 +89,28 @@ function isDSModel(node) {
 }
 
 function isModuleByFilePath(filePath, module) {
-  const fileName = `${module}.js`;
+  if (filePath.indexOf('app/') === -1) { return false; }
+
+  const fileName = `/${module}.js`;
   const folderName = `${module}s`;
```

Comment:
Wouldn't it be a bit more elegant to keep a constant map between a module name and its folder name or use some kind of inflector mechanism, instead of manually tacking on an `s`? I know there won't be other modules, but this still looks tacky.